cpp/test/compat/core/compat.cmake

```diff
@@ -20,3 +20,20 @@ if (CSHARP_COMPAT)
         run (${CSHARP_COMPAT} ${TEST} compat.Compat.json compat.${TEST}.gbc.dat)
     endif()
 endif()
+
+if (JAVA_COMPAT)
+    if (WIN32)
+        set (PATHSEP "\\\;")
+    else()
+        set (PATHSEP ":")
+    endif()
+
+    run (java -classpath ${JAVA_CORE}${PATHSEP}${JAVA_COMPAT} com.microsoft.bond.compat.CompatDriver
```

Comment:
Hmmm... Sounds like we've got some bugs elsewhere in the CMake file. C# codegen and build [works with spaces](https://github.com/Microsoft/bond/tree/e81d51b33b9329642860f02b7fba5733f1c5b96f/cs/test/core/import%20dir%20with%20spaces). Can you open an issue? I'm not able to quickly reproduce the issue on Windows with MSBuild.